dictionaries/condensed-strokes.json

```diff
@@ -1474,8 +1475,8 @@
 "TPOURT/H-PB/TPEUF": "forty-five",
 "TPRA*PBG/EURB": "Frankish",
 "TPRAOEUT/-FL/HREU": "frightfully",
-"TPRAPBG/HREU": "frankly",
 "TPRAPBG/-PBS": "frankness",
+"TPRAPBG/HREU": "frankly",
```

Comment:
@paulfioravanti I'm surprised to see this again after merging this: https://github.com/didoesdigital/steno-dictionaries/pull/295/files#r407178970. Does this need to be rebased or something?

Reply:
Looks like that was the case. Rebase done, and it now looks fixed :+1: